ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
is this a typo? I believe this should not be there.

Reply:
you're right... let me patch this